### PR TITLE
In integrate test add force threshold of 0.0001 and stress threhold 0.001…

### LINKED ISSUE #2844: Fifty integrated test cases failed on Intel env
### Describe Current Status and Possible Solution

The following 50 test cases failed on intel machine.
```
1: 107_PW_outWfcR
1: 107_PW_W90
1: 109_PW_CR_fix_abc
1: 111_PW_elec_add
1: 116_PW_scan_Si2
1: 116_PW_scan_Si2_nspin2
1: 127_PW_15_PK_AF
1: 150_PW_15_CR_VDW3
1: 184_PW_BNDKPAR_SDFT_ALL
1: 184_PW_BNDKPAR_SDFT_MALL
1: 201_NO_KP_DJ_CF_CS_GaAs
1: 201_NO_KP_DJ_Si
1: 208_NO_KP_CS_CR
1: 213_NO_mulliken
1: 215_NO_sol_H2
1: 216_NO_scan_Si2
1: 250_NO_KP_CR_VDW2
1: 250_NO_KP_CR_VDW3
1: 250_NO_KP_CR_VDW3ABC
1: 250_NO_KP_CR_VDW3BJ
1: 281_NO_KP_HSE
1: 282_NO_KP_HSE_complex
1: 283_NO_KP_HF
1: 284_NO_KP_PBE0
1: 285_NO_KP_RE_HSE
1: 286_NO_KP_CR_HSE
1: 283_NO_restart
1: 307_NO_GO_OH
1: 381_NO_GO_S1_HSE
1: 382_NO_GO_S2_HSE
1: 383_NO_GO_SO_HSE
1: 384_NO_GO_S1_HSE_loop0_PU
1: 385_NO_GO_RE_S1_HSE
1: 386_NO_GO_MD_S1_HSE
1: 601_NO_TDDFT_CO_occ
1: 801_PW_LT_sc
1: 802_PW_LT_fcc
1: 803_PW_LT_bcc
1: 804_PW_LT_hexagonal
1: 805_PW_LT_trigonal
1: 806_PW_LT_st
1: 807_PW_LT_bct
1: 808_PW_LT_so
1: 809_PW_LT_baco
1: 810_PW_LT_fco
1: 811_PW_LT_bco
1: 812_PW_LT_sm
1: 813_PW_LT_bacm
1: 814_PW_LT_triclinic
1: 824_NO_LT_fco
```

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Zhuxuegang2022**
…to made less warning output

### Reminder
- [x] Have you linked an issue with this pull request?
- [ ] Have you noticed possible changes of behavior below or in the linked issue?
- [ ] Have you explained the changes of codes in core modules of ESolver, HSolver, ElecState, Hamilt, Operator or Psi? (ignore if not applicable)

### Linked Issue
Fix #2844 

### What's changed?
In integrate test add force threshold of 0.0001 and stress threhold of 0.001. this change will let less "warning" to output.

### CODE REVIEW

**Comment by hongriTianqi** [comment]
Please check to bash script, there might be an error to cause the warning in the picture:
截屏2023-10-23 09 53 22

File: tests/integrate/Autotest.sh [hunk 1]
```diff
@@ -100,16 +104,30 @@ check_out(){
...
 		#--------------------------------------------------
 		if [ ! -n "$deviation" ]; then
-            echo -e "\e[0;31m[ERROR     ] Fatal Error: key $key not found in output.\e[0m"
+			echo -e "\e[0;31m[ERROR     ] Fatal Error: key $key not found in output.\e[0m"
 			let fatal++
 			fatal_case_list+=$dir'\n'
 			break
-        else
+		else
 			if [ $(echo "sqrt($deviation*$deviation) < $threshold"|bc) = 0 ]; then
```
Comment (Zhuxuegang2022) [inline tests/integrate/Autotest.sh:112]: if [ $(echo "sqrt($deviation*$deviation) < $threshold"|bc) = 0  ]; then

File: tests/integrate/Autotest.sh [hunk 2]
```diff
@@ -100,16 +104,30 @@ check_out(){
...
+				if [ $key == "totalforceref" -o $key == "totalstressref"]; then
```
Comment (caic99) [inline tests/integrate/Autotest.sh]:
```suggestion
				if [ $key == "totalforceref" -o $key == "totalstressref" ]; then
```

@Zhuxuegang2022 Actually this if statement is not necessary by "if... elif ... else".

**Comment by caic99** [comment]
@Zhuxuegang2022 #2552 may be solved by remove all warnings in a new PR after merging this one.

**Review by dyzheng — CHANGES_REQUESTED** [review]
please format your code, tabs should be replace by spaces.

**Review by dyzheng — APPROVED** [review]
I agree to set force_threshold and stress_threshold to broaden criterion, because results of different compilers or machines may have small digital errors.

**Pull request merged**